### PR TITLE
#100

### PR DESCRIPTION
**Pull request by shakilsmash**
post er kaj done.

### CODE REVIEW

File: src/main/resources/config/application.yml [hunk 1]
```diff
@@ -6,4 +6,4 @@
 
 spring:
     application:
-        name: kothamala
\ No newline at end of file
+        name: kothamala_rough
```
Comment (ronniegnr) [inline src/main/resources/config/application.yml]: you should not change database name in properties file.

File: src/main/java/com/ronniegnr/kothamala/service/dto/UserDTO.java [hunk 2]
```diff
@@ -9,7 +9,7 @@
 /**
  * A DTO representing a user, with his autorities.
  */
-public class UserDTO {
+public class UserDTO { //data transfer object
```
Comment (ronniegnr) [inline src/main/java/com/ronniegnr/kothamala/service/dto/UserDTO.java]: remove unneccesary comment.

File: src/main/java/com/ronniegnr/kothamala/domain/enumeration/PostStatus.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package com.ronniegnr.kothamala.domain.enumeration;
+
+/**
+ * Created by shaki on 1/10/2017.
```
Comment (ronniegnr) [inline src/main/java/com/ronniegnr/kothamala/domain/enumeration/PostStatus.java]: remove these created by comment.

File: src/main/java/com/ronniegnr/kothamala/domain/User.java [hunk 4]
```diff
@@ -18,6 +18,7 @@
 
     private static final long serialVersionUID = 1;
 
+
```
Comment (ronniegnr) [inline src/main/java/com/ronniegnr/kothamala/domain/User.java]: remove this line break.

File: src/main/java/com/ronniegnr/kothamala/domain/Post.java [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+import java.io.Serializable;
+import java.sql.Timestamp;
+
+@Entity
+@Table(name = "post")
+public class Post extends AbstractAuditingEntity implements Serializable {
+
+    private static final long serialVersionUID = 1;
+
+    private long id;
```
Comment (ronniegnr) [inline src/main/java/com/ronniegnr/kothamala/domain/Post.java]: fix according to this - https://www.codacy.com/app/gnr_ronnie/kothamala/pullRequest?prid=474175

File: src/main/resources/config/application.yml (hunk 1 above)
Comment (shakilsmash) [inline src/main/resources/config/application.yml]: fixed

File: src/main/java/com/ronniegnr/kothamala/service/dto/UserDTO.java (hunk 2 above)
Comment (shakilsmash) [inline src/main/java/com/ronniegnr/kothamala/service/dto/UserDTO.java]: done

File: src/main/java/com/ronniegnr/kothamala/domain/enumeration/PostStatus.java (hunk 3 above)
Comment (shakilsmash) [inline src/main/java/com/ronniegnr/kothamala/domain/enumeration/PostStatus.java]: done

**Pull request merged**